.github/workflows/build-&-publish-docker-image.yml

```diff
@@ -21,14 +20,33 @@ env:
 
 jobs:
   build_and_publish_blobber:
-    runs-on: [self-hosted, build]
+    runs-on: [self-hosted, docker-builds]
     steps:
-      - name: Set GITHUB_ENV
+      - name: Set docker image tag 
         run: |
+          if [[ "${{github.ref}}" == refs/pull/* ]]; then
+            tag=${GITHUB_REF/\/merge/}
+            echo "TAG=$(echo pr-${tag:10})" >> $GITHUB_ENV
+          else
+            echo "TAG=$(([ -z '${{ github.event.pull_request.head.sha }}' ] && echo ${GITHUB_REF#refs/*/} || echo $GITHUB_HEAD_REF) | sed 's/\//-/g')" >> $GITHUB_ENV
```

Comment:
this can be simplified to 
echo "TAG=$(([ echo ${GITHUB_REF#refs/*/} | sed 's/\//-/g')" >> $GITHUB_ENV
because at this point we know we're not in a PR